src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -1333,15 +1333,15 @@ def injectArray(String prefix, def message) {
 def repoFromRequest(String request, String prefix) {
 
     try {
-        def gitMatcher = request =~ /git.+?\/([a-z0-9A-Z_\-]+)(\.git|\?).*/
-        def cliMatcher = request =~ /cli-build.+?\/([a-zA-Z0-9\-_]+)-.*/
+        def gitMatcher = request =~ /git.+?\/([a-z0-9A-Z_\-]+)(?:\.git|\?).*/
```

Comment:
Does this make it so ++ like we saw in that one will work too? Instead of just one +

Reply:
No. I will update it for that. Thanks for the reminder.